diff --git a/src/main/java/meteordevelopment/meteorclient/commands/Command.java b/src/main/java/meteordevelopment/meteorclient/commands/Command.java
index b1ebaf8bc..dd4a663cd 100644
--- a/src/main/java/meteordevelopment/meteorclient/commands/Command.java
+++ b/src/main/java/meteordevelopment/meteorclient/commands/Command.java
@@ -34,8 +34,8 @@ public abstract class Command {
 
     public Command(String name, String description, String... aliases) {
         this.name = name;
-        this.title = Utils.nameToTitle(name);
-        this.description = description;
+        this.title = Utils.tr(Utils.nameToTitle(name));
+        this.description = Utils.tr(description);
         this.aliases = List.of(aliases);
     }
 
