diff --git a/src/main/java/meteordevelopment/meteorclient/renderer/text/Font.java b/src/main/java/meteordevelopment/meteorclient/renderer/text/Font.java
index 7886610..13cff69 100644
--- a/src/main/java/meteordevelopment/meteorclient/renderer/text/Font.java
+++ b/src/main/java/meteordevelopment/meteorclient/renderer/text/Font.java
@@ -29,7 +29,7 @@ public class Font {
     private final Int2ObjectOpenHashMap<CharData> charMap = new Int2ObjectOpenHashMap<>();
-    private static final int size = 2048;
+    private static final int size = 4096;
 
     public Font(ByteBuffer buffer, int height) {
@@ -41,7 +41,8 @@ public class Font {
             STBTTPackedchar.create(128), // Latin Extended-A
             STBTTPackedchar.create(144), // Greek and Coptic
             STBTTPackedchar.create(256), // Cyrillic
-            STBTTPackedchar.create(1) // infinity symbol
+            STBTTPackedchar.create(1), // infinity symbol
+            STBTTPackedchar.create(20992) // CJK Unified Ideographs (U+4E00 - U+9FFF)
         };
 
         // create and initialise packing context
@@ -55,7 +56,8 @@ public class Font {
         packRange.put(STBTTPackRange.create().set(height, 256, null, 128, cdata[2], (byte) 2, (byte) 2));
         packRange.put(STBTTPackRange.create().set(height, 880, null, 144, cdata[3], (byte) 2, (byte) 2));
         packRange.put(STBTTPackRange.create().set(height, 1024, null, 256, cdata[4], (byte) 2, (byte) 2));
-        packRange.put(STBTTPackRange.create().set(height, 8734, null, 1, cdata[5], (byte) 2, (byte) 2)); // lol
+        packRange.put(STBTTPackRange.create().set(height, 8734, null, 1, cdata[5], (byte) 2, (byte) 2));
+        packRange.put(STBTTPackRange.create().set(height, 0x4E00, null, 20992, cdata[6], (byte) 2, (byte) 2));
         packRange.flip();
 
         // write and finish
