diff --git a/src/main/java/meteordevelopment/meteorclient/systems/modules/Module.java b/src/main/java/meteordevelopment/meteorclient/systems/modules/Module.java
index f57182d99..ff2e89c33 100644
--- a/src/main/java/meteordevelopment/meteorclient/systems/modules/Module.java
+++ b/src/main/java/meteordevelopment/meteorclient/systems/modules/Module.java
@@ -57,8 +57,8 @@ public abstract class Module implements ISerializable<Module>, Comparable<Module
         this.mc = Minecraft.getInstance();
         this.category = category;
         this.name = name;
-        this.title = Utils.nameToTitle(name);
-        this.description = description;
+        this.title = Utils.tr(Utils.nameToTitle(name));
+        this.description = Utils.tr(description);
         this.aliases = aliases;
         this.color = Color.fromHsv(Utils.random(0.0, 360.0), 0.35, 1);
 
