diff --git a/src/main/java/meteordevelopment/meteorclient/gui/screens/settings/PacketBoolSettingScreen.java b/src/main/java/meteordevelopment/meteorclient/gui/screens/settings/PacketBoolSettingScreen.java
index 81815a252..abbb97f34 100644
--- a/src/main/java/meteordevelopment/meteorclient/gui/screens/settings/PacketBoolSettingScreen.java
+++ b/src/main/java/meteordevelopment/meteorclient/gui/screens/settings/PacketBoolSettingScreen.java
@@ -12,34 +12,33 @@ import meteordevelopment.meteorclient.settings.PacketListSetting;
 import meteordevelopment.meteorclient.settings.Setting;
 import meteordevelopment.meteorclient.utils.network.PacketUtils;
 import net.minecraft.network.protocol.Packet;
-import net.minecraft.network.protocol.PacketType;
-import org.jetbrains.annotations.NotNull;
 
 import java.util.Set;
 import java.util.function.Predicate;
 
-public class PacketBoolSettingScreen extends CollectionListSettingScreen<PacketType<? extends @NotNull Packet<?>>> {
-    public PacketBoolSettingScreen(GuiTheme theme, Setting<Set<PacketType<? extends @NotNull Packet<?>>>> setting) {
-        super(theme, "Select Packets", setting, setting.get(), PacketUtils.getPackets());
+public class PacketBoolSettingScreen extends CollectionListSettingScreen<Class<? extends Packet<?>>> {
+    public PacketBoolSettingScreen(GuiTheme theme, Setting<Set<Class<? extends Packet<?>>>> setting) {
+        super(theme, "Select Packets", setting, setting.get(), PacketUtils.PACKETS);
     }
 
     @Override
-    protected boolean includeValue(PacketType<? extends @NotNull Packet<?>> value) {
-        Predicate<PacketType<? extends @NotNull Packet<?>>> filter = ((PacketListSetting) setting).filter;
+    protected boolean includeValue(Class<? extends Packet<?>> value) {
+        Predicate<Class<? extends Packet<?>>> filter = ((PacketListSetting) setting).filter;
 
         if (filter == null) return true;
         return filter.test(value);
     }
 
     @Override
-    protected WWidget getValueWidget(PacketType<? extends @NotNull Packet<?>> value) {
-        return theme.label(value.toString());
+    protected WWidget getValueWidget(Class<? extends Packet<?>> value) {
+        return theme.label(PacketUtils.getName(value));
     }
 
     @Override
-    protected String[] getValueNames(PacketType<? extends @NotNull Packet<?>> value) {
+    protected String[] getValueNames(Class<? extends Packet<?>> value) {
         return new String[]{
-            value.toString()
+            PacketUtils.getName(value),
+            value.getSimpleName()
         };
     }
 }
