diff --git a/src/main/java/meteordevelopment/meteorclient/gui/themes/meteor/widgets/WMeteorModule.java b/src/main/java/meteordevelopment/meteorclient/gui/themes/meteor/widgets/WMeteorModule.java
index 1bf15b2..9bcb510 100644
--- a/src/main/java/meteordevelopment/meteorclient/gui/themes/meteor/widgets/WMeteorModule.java
+++ b/src/main/java/meteordevelopment/meteorclient/gui/themes/meteor/widgets/WMeteorModule.java
@@ -10,6 +10,7 @@ import meteordevelopment.meteorclient.gui.themes.meteor.MeteorGuiTheme;
 import meteordevelopment.meteorclient.gui.themes.meteor.MeteorWidget;
 import meteordevelopment.meteorclient.gui.utils.AlignmentX;
 import meteordevelopment.meteorclient.gui.widgets.pressable.WPressable;
+import meteordevelopment.meteorclient.utils.Utils;
 import meteordevelopment.meteorclient.systems.modules.Module;
 import net.minecraft.util.Mth;
 
@@ -29,7 +30,7 @@ public class WMeteorModule extends WPressable implements MeteorWidget {
 
     public WMeteorModule(Module module, String title) {
         this.module = module;
-        this.title = title;
+        this.title = Utils.trIfPresent(title);
         this.tooltip = module.description;
 
         if (module.isActive()) {
